helm/prometheus-rules/templates/alerting-rules/aws.management-cluster.rules.yml

```diff
@@ -103,7 +103,7 @@ spec:
       labels:
         area: kaas
         severity: notify
-        team: se
+        team: phoenix
```

Comment:
Why aren't you using `{{ include "providerTeam" . }}` here as well?

Reply:
These are phoenix specific files (aws and azure) but I think it'd be worth using the include here too as in the future changing responsibilities for different clouds is easier.

Reply:
I've changed these three that you pointed out. We still hardcode phoenix and rocket in almost 200 rules, but the chance of breaking things is too high to do all of them so I'm going to restrict to these changes (`team: se` and existing provider specific team assignment) for now.